Cargo.toml

```diff
@@ -8,6 +8,7 @@ edition = "2018"
 libc = "0.2"
 byteorder = "1"
 log = "0.4"
+epoll = "4.1.0"
```

Comment:
Maybe we can waive this crate? :) Doesn't do a lot (only one file with less than 150 lines of code) but it includes yet another crate in our driver :-/